apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/calendar-links.e2e-spec.ts

```diff
@@ -0,0 +1,180 @@
+      bookingsRepositoryFixture = new BookingsRepositoryFixture(moduleRef);
+      eventTypesRepositoryFixture = new EventTypesRepositoryFixture(moduleRef);
+      oauthClientRepositoryFixture = new OAuthClientRepositoryFixture(moduleRef);
+      teamRepositoryFixture = new TeamRepositoryFixture(moduleRef);
+      schedulesService = moduleRef.get<SchedulesService_2024_04_15>(SchedulesService_2024_04_15);
+
+      organization = await teamRepositoryFixture.create({
+        name: `confirm-bookings-2024-08-13-organization-${randomString()}`,
+      });
+      oAuthClient = await createOAuthClient(organization.id);
+
+      user = await userRepositoryFixture.create({
+        email: userEmail,
+        platformOAuthClients: {
+          connect: {
+            id: oAuthClient.id,
+          },
+        },
+      });
+
+      const userSchedule: CreateScheduleInput_2024_04_15 = {
+        name: `calendar-links-2024-08-13-${randomString()}-schedule`,
+        timeZone: "Europe/Rome",
+        isDefault: true,
+      };
+      await schedulesService.createUserSchedule(user.id, userSchedule);
+      const event = await eventTypesRepositoryFixture.create(
+        {
+          title: `calendar-links-2024-08-13-${randomString()}-event-type`,
+          slug: `calendar-links-2024-08-13-${randomString()}-event-type-slug`,
+          length: 60,
+        },
+        user.id
+      );
+
+      booking = await bookingsRepositoryFixture.create({
+        user: {
+          connect: {
+            id: user.id,
+          },
+        },
+        startTime: new Date(Date.UTC(2020, 0, 8, 12, 0, 0)),
+        endTime: new Date(Date.UTC(2020, 0, 8, 13, 0, 0)),
+        uid: `calendar-links-2024-08-13-booking-${randomString()}`,
+        title: "Test Booking",
+        eventType: {
+          connect: {
+            id: event.id,
+          },
+        },
+        location: "integrations:daily",
+        customInputs: {},
+        metadata: {},
+        attendees: {
+          create: {
+            email: "johndoe@example.com",
+            name: "John Doe",
+            locale: "en",
+            timeZone: "America/New_York",
+          },
+        },
+        responses: {
+          create: {
+            name: "John Doe",
+            email: "johndoe@example.com",
+          },
+        },
+      });
+
+
+      app = moduleRef.createNestApplication();
+      bootstrap(app as NestExpressApplication);
+
+      await app.init();
+    });
+
+    async function createOAuthClient(organizationId: number) {
+      const data = {
+        logo: "logo-url",
+        name: "name",
+        redirectUris: ["http://localhost:5555"],
+        permissions: 32,
+      };
+      const secret = "secret";
+
+      const client = await oauthClientRepositoryFixture.create(organizationId, data, secret);
+      return client;
+    }
+
+    it("should get calendar links for a booking", async () => {
+      const response = await request(app.getHttpServer())
```

Comment:
A very basic test that ensures appropriate results are there through the API.

Detailed tests are unit tests of `getCalendarLinks`